docs/source/en/model_doc/chameleon.md

```diff
@@ -50,13 +50,18 @@ The original code can be found [here](https://github.com/facebookresearch/chamel
 
 - We advise users to use `padding_side="left"` when computing batched generation as it leads to more accurate results. Simply make sure to set `processor.tokenizer.padding_side = "left"` before generating.
 
+- When generating images, we advice users to load the model in `bfloat16` for better results. Simply make sure to set `torch_dtype=torch.bfloat16` when loading the model.
+
 - Note that Chameleon was tuned for safety alignment. If the model is refusing to answer, consider asking a more concrete question, instead of an open question.
 
 - Chameleon generates in chat format which means that the generated text will always be the "assistant's turn". You can enable a text completion generation by passing `return_for_text_completion=True` when calling the processor.
```

Comment:
@zucchini-nlp what do you think of using a chat template instead?

Reply:
I thought about it but Chameleon doesn't have a chat template, and works in chat format by default. It can be switched to text completion by passing `return_text_completion=True` to the processor's call